sys/net/gnrc/routing/rpl/gnrc_rpl_dodag.c

```diff
@@ -40,6 +40,13 @@ static void _rpl_trickle_send_dio(void *args)
 {
     gnrc_rpl_dodag_t *dodag = (gnrc_rpl_dodag_t *) args;
     ipv6_addr_t all_RPL_nodes = GNRC_RPL_ALL_NODES_ADDR;
+
+    /* a leaf node does not send DIOs periodically */
```

Comment:
I'm referring to the pre-last paragraph of https://tools.ietf.org/html/rfc6550#section-8.5